src/Validation/Validation.js

```diff
@@ -4,9 +4,13 @@ export function checkEmail(email) {
   return reg_email.test(email);
 }
 
-export function checkPwd(pwd) {
+export function checkPwd(password) {
   const reg_pwd =
     /^.*(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
-
-  return reg_pwd.test(pwd);
+  return reg_pwd.test(password);
 }
+
+export const validator = {
+  email: email => checkEmail(email),
+  password: password => checkPwd(password),
```

Comment:
```suggestion
  email: checkEmail
  password: checkPwd
```
굳이 새로운 함수를 만들 필요 없이, checkEmail, checkPwd 함수 그대로 넣어줘도 될 것 같습니다!

Reply:
넵!